apps/fluent-tester/src/FluentTester/TestComponents/Button/ButtonIconTest.tsx

```diff
@@ -41,6 +41,14 @@ export const ButtonIconTest: React.FunctionComponent = () => {
             tooltip="button tooltip"
           />
         ) : null}
+        <Text>Right Button icon</Text>
```

Comment:
nit: Change to "end" to align with the prop name?

Reply:
fixed